generateEntityFromLeesStore.py

```diff
@@ -73,13 +73,13 @@
 
 ############  USER INPUT ###########
 # the root directory of your app that contains the angular and aspnet-core folders
-target_dir = "C:\\MyProjects\\Sample\\5.6.0\\"
+target_dir = "..\\"
 
 # the namespace of your solution
-solutionName = "Sample"
+solutionName = "LeesStore"
```

Comment:
It'd be really nice if this script could calculate this value.  In the meantime I feel that LeesStore is an improvement, especially if this were to be a branch or PR off of the LeesStore repo.